web/src/main/java/org/openmrs/web/servlet/LoginServlet.java

```diff
@@ -59,6 +59,17 @@
 	private Map<String, Date> lockoutDateByIP = new HashMap<String, Date>();
 	
 	/**
+	 * @see javax.servlet.http.HttpServlet#doGet(javax.servlet.http.HttpServletRequest,
+	 *      javax.servlet.http.HttpServletResponse)
+	 */
+	@Override
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		if(Context.getAuthenticatedUser() != null) {
```

Comment:
Yes right. I'll do that. But that was being done in a controller and we added model attributes there when privi. Should we add a LoginFormController here then?